<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="FTR3COF93BK8A76" name="testcases.Voice_test" tests="1" failures="1" timestamp="7 Jul 2021 02:38:45 GMT" time="129.923" errors="0">
  <testcase name="Voice_Char_Test" time="129.923" classname="testcases.Voice_test">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [148] line_end; expected [Good. What seems to be the trouble with that telephone number?1. dial tone problem2. Voicemail3. Can&#039;t Call Out4. Caller ID Issue5. Can&#039;t Be Called6. Feature Issue7. Transmission Problem8. Data Problem9. Outside damage10. 911 Related Issue11. Miscellaneous Issue] but found [Good. What seems to be the trouble with that telephone number?1. dial tone problem2. Voicemail3. Can&#039;t Call Out4. Caller ID Issue5. Can&#039;t Be Called6. Features7. Transmission8. Data Problem9. outside equipment problem10. 911 Related Issue11. Miscellaneous],
	@@line_start [153] line_end; expected [You can say things like, no dial tone, I can&#039;t make calls, receive calls, problem with a phone feature, or a phone jack, or a telephone pole outside, etc.
1. dial tone problem
2. Voicemail
3. Can&#039;t Call Out
4. Caller ID Issue
5. Can&#039;t Be Called\n6. Feature Issue
7. Transmission Problem
8. Data Problem
9. Outside damage
10. 911 Related Issue
11. Miscellaneous Issue
] but found [You can say things like, no dial tone, I can&#039;t make calls, receive calls, problem with a phone feature, or a phone jack, or a telephone pole outside, etc.
1. dial tone problem
2. Voicemail
3. Can&#039;t Call Out
4. Caller ID Issue
5. Can&#039;t Be Called
6. Features
7. Transmission
8. Data Problem
9. outside equipment problem
10. 911 Related Issue
11. Miscellaneous
],
	@@line_start [158] line_end; expected [Is it dial tone problem?] but found [I see. Is it dial tone problem?]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [148] line_end; expected [Good. What seems to be the trouble with that telephone number?1. dial tone problem2. Voicemail3. Can't Call Out4. Caller ID Issue5. Can't Be Called6. Feature Issue7. Transmission Problem8. Data Problem9. Outside damage10. 911 Related Issue11. Miscellaneous Issue] but found [Good. What seems to be the trouble with that telephone number?1. dial tone problem2. Voicemail3. Can't Call Out4. Caller ID Issue5. Can't Be Called6. Features7. Transmission8. Data Problem9. outside equipment problem10. 911 Related Issue11. Miscellaneous],
	@@line_start [153] line_end; expected [You can say things like, no dial tone, I can't make calls, receive calls, problem with a phone feature, or a phone jack, or a telephone pole outside, etc.
1. dial tone problem
2. Voicemail
3. Can't Call Out
4. Caller ID Issue
5. Can't Be Called\n6. Feature Issue
7. Transmission Problem
8. Data Problem
9. Outside damage
10. 911 Related Issue
11. Miscellaneous Issue
] but found [You can say things like, no dial tone, I can't make calls, receive calls, problem with a phone feature, or a phone jack, or a telephone pole outside, etc.
1. dial tone problem
2. Voicemail
3. Can't Call Out
4. Caller ID Issue
5. Can't Be Called
6. Features
7. Transmission
8. Data Problem
9. outside equipment problem
10. 911 Related Issue
11. Miscellaneous
],
	@@line_start [158] line_end; expected [Is it dial tone problem?] but found [I see. Is it dial tone problem?]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.Voice_test.Voice_Char_Test(Voice_test.java:205)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- Voice_Char_Test -->
</testsuite> <!-- testcases.Voice_test -->
